pkg/osquery/tables/dev_table_tooling/table.go

```diff
@@ -54,17 +55,21 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 			continue
 		}
 
-		output, err := tablehelpers.Exec(ctx, t.logger, 30, cmd.binPaths, cmd.args, false)
-		if err != nil {
+		result := map[string]string{
+			"name":   name,
+			"args":   strings.Join(cmd.args, " "),
+			"output": "",
+			"error":  "",
+		}
+
+		if output, err := tablehelpers.Exec(ctx, t.logger, 30, cmd.binPaths, cmd.args, false); err != nil {
 			level.Info(t.logger).Log("msg", "execution failed", "name", name, "err", err)
-			continue
+			result["error"] = err.Error()
+		} else {
```

Comment:
You could strike the `if` here, but this is fine too.

Reply:
I think I'm wrong, at least, sometimes. If err is nil, some ways of casting it to a string work, and some fail. So probably best to keep it like this